### PR TITLE
Update obsolete Brave info (pkg on macOS)

### PR DESCRIPTION
**Pull request by fmarier**
Changes proposed in this PR:

- Remove [obsolete](https://github.com/brave/brave-browser/issues/32316) warning about PKG on macOS
- ~~Remove anti-fingerprinting mode which [no longer exists](https://brave.com/privacy-updates/28-sunsetting-strict-fingerprinting-mode/)~~

Conflict of interest statement: I work for Brave Software.

<summary>

- [x] I agree to the terms listed below:
  <details><summary>Contribution terms (click to expand)</summary>
  1) I am the sole author of this work.
  2) I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
  3) I have disclosed any relevant conflicts of interest in my post.
  4) I agree to the Community Code of Conduct.
  </details>

### CODE REVIEW

**Comment by dngray** [comment]
I'm curious to know what version of Brave you have, because this option still seems to be in the Linux version https://github.com/privacyguides/privacyguides.org/pull/2641#pullrequestreview-2187368837

**Comment by fmarier** [comment]
> I'm curious to know what version of Brave you have, because this option still seems to be in the Linux version [#2641 (review)](https://github.com/privacyguides/privacyguides.org/pull/2641#pullrequestreview-2187368837)

Latest version in the official .deb repo:
```
Brave	1.67.123 Chromium: 126.0.6478.126 (Official build) (64 bits) 
Revision	fc7619ef621fe4e6a0fa8b16c718f78ffc97a861
OS	Linux
```

**Comment by fmarier** [comment]
I'll rebase this once https://github.com/privacyguides/privacyguides.org/pull/2641 lands to remove the fingerprinting commit.

File: docs/desktop-browsers.md [hunk 1]
```diff
@@ -288,7 +286,6 @@ Brave allows you to select additional content filters within the internal `brave
 
 - [x] Select **Strict** under *Upgrade connections to HTTPS*
 - [x] (Optional) Select **Block Scripts** (1)
-- [x] Select **Strict, may break sites** under *Block fingerprinting*
 - [x] Check **Forget me when I close this site** (2)
```
Comment (jonaharagon) [inline docs/desktop-browsers.md:289]: This setting does still exist as a boolean.

```suggestion
- [x] Check **Block fingerprinting**
- [x] Check **Forget me when I close this site** (2)
```

**Review by jonaharagon — CHANGES_REQUESTED** [review]
Thanks for the PR! Dang, glad they made this change to macOS downloads.

File: docs/desktop-browsers.md (hunk 1 above)
Comment (jonaharagon) [reply docs/desktop-browsers.md:289]: Oh I see this was separately proposed in #2641, ignore :)

**Comment by dngray** [comment]
> Latest version in the official .deb repo:

That's interesting because I was using the latest one from the fedora repo in the release channel. Version 1.67.134 Chromium: 126.0.6478.186 (Official Build) (64-bit). I installed beta 1.69.117-1, now shows Version 1.69.117 Chromium: 127.0.6533.57 (Official Build) beta (64-bit) and it still has that option as strict.

**Review by jonaharagon — COMMENTED** [review]
If you wouldn't mind rebasing then I'll approve :)

**Comment by fmarier** [comment]
@jonaharagon Rebased.

**Review by jonaharagon — APPROVED** [review]
LGTM thank you!

**Comment by fmarier** [comment]
> That's interesting because I was using the latest one from the fedora repo in the release channel. Version 1.67.134 Chromium: 126.0.6478.186 (Official Build) (64-bit). I installed beta 1.69.117-1, now shows Version 1.69.117 Chromium: 127.0.6533.57 (Official Build) beta (64-bit) and it still has that option as strict.

Even after you restart the browser? If so, are you blocking https://variations.brave.com/ by any chance? That's where the flag settings file (called a `seed`) is downloaded from.  If that doesn't get downloaded for you, then you're not getting the intended configuration. You may be behind on some features we're only enabling via that mechanism.

**Review by kimg45 — APPROVED** [review]

**Pull request merged**